### PR TITLE
e2e: add e2e test for render command

### PR DESCRIPTION
**Pull request by pratikjagrut**
**What issue type does this pull request address?** (keep at least one, remove the others) 
/kind test

**What does this pull request do? Which issues does it resolve?** (use `resolves #<issue_number>` if possible) 
resolves #
Add e2e test for render command

### CODE REVIEW

File: e2e/tests/render/render.go [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+		// create build command
+		renderCmd := &cmd.RenderCmd{
+			GlobalFlags: &flags.GlobalFlags{
+				NoWarn: true,
+			},
+			SkipPush: true,
+			Writer:   stdout,
+		}
+		err = renderCmd.Run(f)
+		framework.ExpectNoError(err)
```
Comment (FabianKramm) [inline e2e/tests/render/render.go:46]: Would be great if you check the rendered yamls as well, you can use the `--silent` flag for this

File: e2e/tests/render/render.go [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+		stdout := &Buffer{}
+		// create build command
+		renderCmd := &cmd.RenderCmd{
+			GlobalFlags: &flags.GlobalFlags{
+				NoWarn: true,
+			},
+			SkipPush: true,
+			Writer:   stdout,
+		}
+		err = renderCmd.Run(f)
```
Comment (FabianKramm) [inline e2e/tests/render/render.go:63]: Would be great if you check the rendered yamls as well

**Review by FabianKramm — CHANGES_REQUESTED** [review]
@pratikjagrut thanks for the PR! Please see my comments

File: e2e/tests/render/render.go (hunk 1 above)
Comment (pratikjagrut) [reply e2e/tests/render/render.go:48]: done

**Review by FabianKramm — APPROVED** [review]
LGTM!

**Pull request merged**